pgmpy/readwrite/XMLBIF.py

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env python
 
+import string
+import re
+from io import BytesIO
+
+# TODO input and output state
```

Comment:
@finnhacks42 Look for state name decorator file in utils. State name decorator adds in an extra layer that take care of mapping of variables.